### DISCUSSION TITLE
EJBCA7 and mysql 8

### DISCUSSION THREAD

**Discussion by syisunny** [Keyfactor/ejbca-ce#335] [Q&A]
Hello,
We would like to deploy ejbca 7 with MySQL 8. does anyone have this experience? is there any doc for it?
Best regards,
Yi SONG

**Comment by primetomas** [comment]
You can find installation instructions here:
https://doc.primekey.com/ejbca/ejbca-installation

If you want to use the container (easier to deploy) with an external database you can find documentation here:
https://hub.docker.com/r/keyfactor/ejbca-ce
